### PR TITLE
Capstone home and about page

### PR DESCRIPTION
**Pull request by Osoro254Alex**
These are the things that I did on Capstone-Project-Concert --Page Repo:

- Create a Feature Branch.
- Add index.html file.
- Add style.css file.
- Add Javascript file.
- Add a descriptive readme file.
- Add gitignore on node modules.
- Add images folder.
- Render the speakers dynamically.
- Add functionality to the menu icon using Javascript.
- Build a separate About page.
- Install and fix the Linter error.
- Push to Remote
- Record a video and Add it to READme.MD file.
- Deploy the Pages in Git and add the link in READme.md file.

## Loom Video Link 
- [Video-Link](https://www.loom.com/share/1d83f49e01d0419f9f48e607c0dd8d93)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      </div>
+    </section>
+    <article>
+      <section class="sec1">
+        <p class="para1">
+          "Come One Come All"
+        </p>
+        <div class="title">
+          <h1 class="main-title">
+            <span>CULTURAL</span> MUSIC AFRICA CONCERT 2021
```
Comment (Lucas-Masaba) [inline index.html:73]: - [ ] Please make this text transparent like in the original design.
### Your Implementation
image
### Required
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          </h1>
+        </div>
+        <P class="para2">
+          Music is an essential element of culture. Sharing music from one culture to another gives people an insight into another way of life. All are invited from all countries in taking places in June, in Kenya
+        </P>
```
Comment (Lucas-Masaba) [inline index.html:78]: - [ ] Please put a white border around this paragraph like in the original design.
### Your Implementation
image

### Required
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+        <h2 class=" year">
+          2021.06.20(FRI) ~ 21(SAT)
+        </h2>
+        <p class="para2 differ">
+          @Nyayo Stadium in Kenya, Cultural center and more
+        </p>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please adjust the font color and size to look familiar to that of the original design.
### Your Implementation
image
### Required
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+        </div>
+      </section>
+      <section id="guests" class="sec3">
+        <h2 class="sec-2-title">
+          Guest of Honour
+        </h2>
+        <hr class="line">
+        <ul class="speakers">
+
+        </ul>
```
Comment (Lucas-Masaba) [inline index.html:154]: - [ ] Please add padding to the left and right to make the display of the speakers look like that of the original design.
### Your Implementation
image
### Required
image

File: index-about.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+              mcconcert2021@mckenya.org <span>&GT;</span>
+            </b>
+          </a>
+        </p>
+        <h2 class="about-title">
+          Cultural Music Africa Concert 2021 Logo
+        </h2>
+        <p class="para2">
+          The logo of MC Music Culture 2021 was decided through the logo competition from 9, April to 10 May.
+        </p>
```
Comment (Lucas-Masaba) [inline index-about.html]: - [ ] Please add the background image as seen in the original design.
### Your Implementation
image
### Required
image

- [ ] Add all the necessary background colours and designs as that of the original design for this block of the about page.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Osoro254Alex  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@Lucas-Masaba 
To put background image in the about.
To increase font weight on header.
To add padding speakers section

**Review by Ismailco — APPROVED** [review]
### Hi @Osoro254Alex, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

cr-approve

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by Osoro254Alex** [comment]
@Ismailco 
Approved the changes that I made as per the previous reviewer @Lucas-Masaba

**Pull request merged**